distribution/test/itests/test-itests-ddf/src/test/resources/customtypedefinitions.json

```diff
@@ -0,0 +1,24 @@
+{
+  "metacardTypes": [
+    {
```

Comment:
This filename might be easier to read as `custom-type-definitions.json` or camel case